benchmarks/scala-dotty/src/main/scala/org/renaissance/scala/dotty/Dotty.scala

```diff
@@ -80,12 +80,34 @@ class Dotty extends RenaissanceBenchmark {
   private val DOTTY_ARG_TYPE_CONVERSION = "-language:implicitConversions"
 
   override def runIteration(c: Config): BenchmarkResult = {
+    /*
+     * Construct the classpath for the compiler. Unfortunately, Dotty is
+     * unable to use current classloader (either of this class or this
+     * thread) and thus we have to explicitly pass it. Note that
+     * -usejavacp would not work here as that reads from java.class.path
+     * property and we do not want to modify global properties here.
+     *
+     * Therefore, we leverage the fact that we know that our classloader
+     * is actually a URLClassLoader that loads the benchmark JARs
+     * from temporary directory. And we convert all the URLs to
+     * plain file paths.
+     *
+     * Note that using URLs as-is cannot be used as that prepends the
+     * "file:/" protocol that is not handled well on Windows when
+     * on classpath.
+     *
+     * Note that it would be best to pass the classloader to the compiler
+     * but that seems to be impossible with current API (see discussion
```

Comment:
Have you found anything about whether this may change in the future?

Reply:
No, but I did only a quick search through the issues that are on GitHub.